src/sysl/util/file.py

```diff
@@ -22,3 +23,10 @@ def java(self, w, name, package):
                                            ' (check BUILD has sysl_model(..., entities=[..., "' + name + ', ...], ...))')
             self.entities.remove(name)
         self(w, os.path.join(self.package.replace('.', '/'), name + u'.java'))
+
+
+def areFilesIdentical(fname1, fname2):
```

Comment:
This function should probably live with a utility module in the tests top-level directory.
However that would require for tests to have a package structure and be on the python path which I wanted to avoid.

Reply:
It's fine where it is, but it should be called `filesAreIdentical`. Predicate-style functions should be phrased as logical propositions, not as questions.

Reply:
Done